src/codeu/chat/common/Writeable.java

```diff
@@ -14,6 +14,7 @@
   public static final String USER_STR = "user";
   public static final String MESSAGE_STR = "message";
   public static final String CONVERSATION_STR = "conversationheader";
+  public static final String CHANGE_ACCESS_REQUEST_STR = "changeaccessrequest";
```

Comment:
in retrospect, these could have probably been enum values. you don't necessarily have to change it though.